### PR TITLE
Enable style checks

### LINKED ISSUE #250: Enable style check (and other tests) on jcsda_emc_spack_stack branch
We should enable the default Spack checks on our branch

### PR DESCRIPTION
**Pull request by kgerheiser**
Fix https://github.com/NOAA-EMC/spack-stack/issues/250

### CODE REVIEW

File: var/spack/repos/jcsda-emc/packages/shumlib/package.py [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-    #version('2021.07.1', commit='a4ea525ad3bf04684ef39b0241991a350e2b7241')
-    #version('2021.03.1', commit='58f599ce9cfb4bd47197125548a44039695fa7f1')
-    #version('2020.11.1', commit='58f599ce9cfb4bd47197125548a44039695fa7f1')
+    # version('2021.10.1', commit='545874fba961deadf4b2758926be7c26f4c8dcb9')
+    # version('2021.07.1', commit='a4ea525ad3bf04684ef39b0241991a350e2b7241')
+    # version('2021.03.1', commit='58f599ce9cfb4bd47197125548a44039695fa7f1')
+    # version('2020.11.1', commit='58f599ce9cfb4bd47197125548a44039695fa7f1')
 
     def edit(self, spec, prefix):
-        env['LIBDIR_OUT'] = os.path.join(os.path.join(self.build_directory, 'spack-build'))
```
Comment (kgerheiser) [inline var/spack/repos/jcsda-emc/packages/shumlib/package.py:29]: @climbfuji is the double `os.path.join` necessary here? It looked like it was doing nothing so I cut it to correct the line length.

Comment (climbfuji) [reply var/spack/repos/jcsda-emc/packages/shumlib/package.py:29]: Certainly not needed.

**Review by climbfuji — APPROVED** [review]
Thanks for making all these changes!

**Comment by kgerheiser** [comment]
The only file not passing the style checks is `meta_modules.py` and it's a pain to fix so I left it. When the style checks are run in a normal PR it only checks files that have changed.

**Comment by climbfuji** [comment]
> The only file not passing the style checks is `meta_modules.py` and it's a pain to fix so I left it. When the style checks are run in a normal PR it only checks files that have changed.

We definitely have to rewrite `meta_modules.py` at some point sooner than later. That was supposed to be a first, quick solution ...

**Pull request merged**